config/menus/glue.cfg

```diff
@@ -359,3 +359,32 @@ looplist i [
     ui_checkbox2 ui_edit_button ui_edit_button2 ui_edit_checbox ui_edit_checbox2
 ] [setcomplete $i 1]
 
+get_server_count            = [ result (getserver) ]
```

Comment:
Naming inconsistency. This one uses `get_server`, but the rest is `getserver`. I'd really just go for `get_server`, I guess, easier to read and understand.

Reply:
done.